presto-parser/src/main/java/com/facebook/presto/sql/tree/RoutineCharacteristics.java

```diff
@@ -0,0 +1,124 @@
+/*
```

Comment:
A general comment, not specific to this - for things like this, it may be simpler to just add these as attributes to the CreateFunction node to reduce code bloat and make it easy to work with the visitor.

Reply:
This will be a reusable unit in `ALTER FUNCTION` so it makes sense to have it separated. Visitor won't touch this class since it is not a subclass of `Node`, or a child of `Create Function`.